pkg/apis/storage/types.go

```diff
@@ -337,6 +337,35 @@ type CSIDriverSpec struct {
 	//
 	// +optional
 	StorageCapacity *bool
+
+	// ServiceAccountTokens indicates the csi driver needs pods' service account
+	// tokens it is mounting volume for to do necessary authentication. Kubelet
+	// will pass the tokens in VolumeContext in the CSI NodePublishVolume calls.
+	// The CSI driver should parse and validate the following VolumeConext:
+	// "csi.storage.k8s.io/serviceAccount.tokens": {
+	//   'audience-0': {
```

Comment:
Is "audience-0" actually the name of the key, or is it some other value?

Reply:
it is a value that user specifies.

Reply:
Can we add some comments to which fields are fixed strings vs which are going to depend on user provider values? Maybe we can use the "<foo>" notation to denote which parts are going to be variable.

Reply:
done